blog/views.py

```diff
@@ -52,3 +49,12 @@ class BlogDayArchiveView(BlogViewMixin, DayArchiveView):
 
 class BlogDateDetailView(BlogViewMixin, DateDetailView):
     banner_is_title = False
+
+    def get_queryset(self):
+        """Allows staff users with blog write permission to view unpublished entries"""
+        if self.request.user.is_staff and self.request.user.has_perm(
+            "blog.change_entry"
+        ):
+            return Entry.objects.all()
+        else:
+            return Entry.objects.published()
```

Comment:
```suggestion
            return self.get_queryset()
```
I would reuse `get_queryset` from `BlogViewMixin`.

Reply:
Ok, that's fair :+1: